### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by flbulgarelli**

### CODE REVIEW

File: lib/test_runner.rb [hunk 1]
```diff
@@ -31,21 +28,17 @@ def gobstones_path
...
-      end
-    ensure
-      [@html_output_file, @actual_final_board_file].each { |it| it.close }
-      [@html_output_file, @actual_final_board_file, @source_file, @initial_board_file].each { |it| it.unlink }
+    if status == :passed
+      compute_test_status
+    else
+      [get_error_message(result), status]
     end
+  ensure
```
Comment (faloi) [inline lib/test_runner.rb]: Nice trick, an ensure block doesn't need a begin clause...

File: lib/test_compiler.rb [hunk 2]
```diff
@@ -2,13 +2,13 @@
 require 'yaml'
 
 class TestCompiler
-  def compile(test_src, extra_src, content_src)
+  def create_compilation_file!(test_src, extra_src, content_src)
```
Comment (faloi) [inline lib/test_compiler.rb]: +1 to this, we should rename the method on Mumukit to improve its semantic

Comment (flbulgarelli) [inline lib/test_compiler.rb]: Yes. Do you want to add an issue?

Comment (faloi) [inline lib/test_compiler.rb]: [Done](https://github.com/mumuki/mumukit/issues/5)

**Pull request merged**